tests/adapter/dbt/tests/adapter/utils/fixture_array_append.py

```diff
@@ -0,0 +1,12 @@
+# array_append
+
+models__array_append_expected_sql = """
+select 1 as id, {{ array_construct([1,2,3,4]) }} as array_col union all
+select 2 as id, {{ array_construct([4]) }} as array_col
+""".lstrip()
```

Comment:
what is the `.lstrip()` for?

Reply:
Good question! In this particular case, it isn't needed, and I can remove it :)

More detail:
`.lstrip()` strips off the leading whitespace so that there isn't an empty newline at the beginning of this "file" (string that ends up being materialized as a file for pytest'ing purposes).

It's useful for seed CSV "files" in particular because it allows the text to be more readable. You can see examples with `git grep "lstrip" ./`.

Reply:
thanks for the explanation!